_includes/popup.html

```diff
@@ -3,8 +3,8 @@
     <div class="picture">
     {% if include.person.image %}
     {% if include.who == 'student' %}
-    <img src="images/students/{{include.person.image}}"
-    alt="{{include.person.name}}"
```

Comment:
Dont add yourself statically into the index.html file make use of Jekyll. The current convention is to insert your name into the students.yml file and place your image in 240x240 pixels in the images/students folder.